biome.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.8.2/schema.json",
+  "organizeImports": {
+    "enabled": true
+  },
+
+  "files": {
+    "ignore": ["package.json", "bun.lockb", "*.md"]
+  },
+
+  "formatter": {
+    "indentWidth": 2,
+    "indentStyle": "space",
+    "lineWidth": 80
+  },
```

Comment:
we can use the default formatter settings, which is `tab` as the separator. It will take effect if you delete the formatter block.